livekit_ingress.proto

```diff
@@ -41,20 +41,46 @@ enum IngressInput {
 message IngressAudioOptions {
     string name = 1;
     TrackSource source = 2;
+	oneof encoding_options {
+        IngressAudioEncodingOptionsPreset preset = 3;
+        IngressAudioEncodingOptions advanced = 4;
+    }
+}
+
+message IngressVideoOptions {
+    string name = 1;
+    TrackSource source = 2;
+	oneof encoding_options {
+        IngressVideoEncodingOptionsPreset preset = 3;
+        IngressVideoEncodingOptions advanced = 4;
+    }
+}
+
+enum IngressAudioEncodingOptionsPreset {
+    OPUS_STEREO_96KBPS = 0;      // OPUS, 2 channels, 96kbps
+//    OPUS_MONO_64KBS = 1;         // OPUS, 1 channel, 64kbps
+}
+
+enum IngressVideoEncodingOptionsPreset {
+    H264_720P_30FPS_3_LAYERS = 0;       // 1280x720,  30fps, 3000kbps main layer, 3 layers total
```

Comment:
I think we've used lower bitrate than this in client SDKs.. would be great to make consistent tho.